index.html

```diff
@@ -198,6 +198,30 @@ <h3>Expression Format</h3>
           [=preference=] for the Global Privacy Control value.
         </p>
       </section>
+      <section>
+        <h3>Preference Caching</h3>
+        <p>
+          The [=preference=] MUST be cached on each top-level navigation to ensure consistency in communication of
+          the person's request that their data "not be sold or shared." This means that if the [=preference=] changes
+          during or after a top-level navigation, it WILL NOT be reflected until the next navigation.
+        </p>
+        <p>
+          A [=top-level browsing context=] has a <dfn><code>gpcAtNavigation</code></dfn> boolean.
+          It is initially <code>false</code>.
+        </p>
+        <p>
+          When [=process a navigate fetch=] is run with [=top-level browsing context=] as <code>browsingContext</code>,
+          [=top-level browsing context=]'s <dfn><code>gpcAtNavigation</code></dfn> is updated to
+          <code>true</code> if the person's  [=preference=] is enabled, and <code>false</code> if
+          the person's [=preference=] is disabled or has not been set.
+        </p>
+        <p>
+          If [=preference=] is changed to be inconsistent with some <code>gpcAtNavigation</code> cached in a
+          [=top-level browsing context=], the user agent SHOULD inform the user of any inconsistent tabs and provide
+          the option to reload them, refreshing the cached <code>gpcAtNavigation</code> to reflect the current [=preference=]
+          The user agent MAY prompt for a refresh of all inconsistent tabs at once or for each individually as the user interacts with them.
```

Comment:
Not sure this second sentence is additive/necessary, but not a problem either way.